source-code-flow-manager-gitlab/src/test/java/com/iprody/source/code/flow/manager/api/job/PipelineJobAPITest.java

```diff
@@ -0,0 +1,68 @@
+package com.iprody.source.code.flow.manager.api.job;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.iprody.source.code.flow.manager.api.job.domain.Job;
+import okhttp3.mockwebserver.MockResponse;
+import okhttp3.mockwebserver.MockWebServer;
+import okhttp3.mockwebserver.RecordedRequest;
+import org.junit.jupiter.api.*;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.MediaType;
+import org.springframework.web.reactive.function.client.WebClient;
+import reactor.core.publisher.Flux;
+import reactor.test.StepVerifier;
+
+import java.io.IOException;
+
+class PipelineJobAPITest {
+    private static MockWebServer mockBackEnd;
+    private static ObjectMapper objectMapper;
+    private static PipelineJobAPI jobAPI;
+
+    @BeforeAll
+    static void setUp() throws IOException{
+        mockBackEnd = new MockWebServer();
+        mockBackEnd.start();
+        objectMapper = new ObjectMapper();
+    }
+
+    @AfterAll
+    static void tearDown() throws IOException {
+        mockBackEnd.shutdown();
+    }
+
+    @BeforeEach
+    void initialize() {
+        String baseUrl = String.format("http://localhost:%s", mockBackEnd.getPort());
+
+        WebClient client = WebClient.builder()
+                .baseUrl(baseUrl)
+                .defaultHeader(HttpHeaders.CONTENT_TYPE, MediaType.APPLICATION_JSON_VALUE)
+                .build();
+
+        jobAPI = new PipelineJobAPI(client);
+    }
+
+    @Test
+    void getPipelineJobs() throws JsonProcessingException, InterruptedException{
```

Comment:
1. Improvement 
```suggestion
    @SneakyThrows
    void getPipelineJobs()
```
2. What this method tests? Try to use approach `Given, When, Then` for naming a test method
3. Bad scenario is missing (http errors 4XX, 5XX)